src/main/scala/beam/agentsim/events/PathTraversalEvent.scala

```diff
@@ -0,0 +1,141 @@
+package beam.agentsim.events
+
+import java.util
+import java.util.Collections
+import java.util.concurrent.atomic.AtomicReference
+
+import beam.agentsim.agents.vehicles.BeamVehicleType
+import beam.router.Modes.BeamMode
+import beam.router.model.BeamLeg
+import org.matsim.api.core.v01.Id
+import org.matsim.api.core.v01.events.Event
+import org.matsim.vehicles.Vehicle
+
+import collection.JavaConverters._
+
+class PathTraversalEvent(
+  val time: Double,
+  val vehicleId: Id[Vehicle],
+  val driverId: String,
+  val vehicleType: String,
+  val seatingCapacity: Int,
+  val standingRoomCapacity: Int,
+  val fuelType: String,
+  val numPass: Int,
```

Comment:
Can we name this for what it is - `numberOfPassengers` (or `numPassengers` at least)? And then clean up the other usages to be specific.